lib/atlas/dataset/derived_dataset.rb

```diff
@@ -4,7 +4,7 @@ class Dataset::DerivedDataset < Dataset
 
     attribute :base_dataset, String
     attribute :scaling,      Preset::Scaling
-    attribute :init,         Hash[Symbol => Float]
+    attribute :init,         Hash[InitializerInput => Float]
```

Comment:
> Does that feel "clunky" and awkward to you?

Well, yes and no. It makes sense but the naming might be a bit confusing at first sight. Just as with the `Node#(in_out)slots` when I first looked at that.